wallet-core/rpc-requirements.md

```diff
@@ -0,0 +1,40 @@
+# RPC / API Requirements
+
+Trust Wallet is a light client wallet, which means it needs to get data / information from remote nodes (either light nodes or full nodes). In order to integrate your blockchain into Trust Wallet smoothly, you must also fulfill RPC / API requirements. We list all needed APIs or RPCs plus an example below for your reference.
+
+## for Trust Wallet: 
+
+We need API or RPC to: 
+
+- [ ] query account / address balance
+- [ ] query transaction details
+- [ ] query fee / nonce for sending transaction
+- [ ] query blockchain status (block height etc)
+- [ ] send raw transaction
+
+## for blockatlas: 
+
+We need API or RPC to: 
+
+- [ ] query sent / received transactions for an account / address
```

Comment:
link to blockatlas: `https://developer.trustwallet.com/blockatlas/newblockchain`

Reply:
Added to above